resteasy-core/src/main/java/org/jboss/resteasy/plugins/providers/MultiValuedArrayParamConverter.java

```diff
@@ -0,0 +1,189 @@
+package org.jboss.resteasy.plugins.providers;
+
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Array;
+import java.lang.reflect.Method;
+
+import javax.annotation.Priority;
+import javax.ws.rs.ext.ParamConverter;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.resteasy.core.StringParameterInjector;
+import org.jboss.resteasy.resteasy_jaxrs.i18n.Messages;
+
+/**
+ * @author Marek Kopecky mkopecky@redhat.com
+ * @author Ron Sigal rsigal@redhat.com
+ */
+@Provider
+@Priority(Integer.MAX_VALUE)
+public class MultiValuedArrayParamConverter implements ParamConverter<Object>
```

Comment:
What about ParamConverter<Object[]> here instead of ParamConverter<Object> ?

Reply:
Please forget about this comment. 
As said below, I get the point of the java disctinction between primitive array and object array :+1: